pipeline/trusted-resources/pkg/trustedtask/verify.go

```diff
@@ -33,15 +33,15 @@ import (
 
 const (
 	signingConfigMap = "config-trusted-resources"
-	kmsAnnotation    = "tekton.dev/kms"
+	KmsAnnotation    = "tekton.dev/kms"
```

Comment:
nit: 
```suggestion
	KMSAnnotation    = "tekton.dev/kms"
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Thanks! 😄